packages/terser/index.js

```diff
@@ -26,6 +26,45 @@ function isDirectory(input) {
   return fs.lstatSync(path.join(process.cwd(), input)).isDirectory();
 }
 
+// Returns a single quotes version of str
+function singleQuotes(str) {
+  return `'${str.replace(/'/g, '').replace(/"/g, '')}'`;
+}
+
+// Ensures that args are well formed.
+// Work-around for an issue on Windows when exec bin path is not quoted.
+// In --source-map, base=bazel-out/x64_windows-opt-exec-2B5CBBC6/bin must
```

Comment:
interesting we didn't have this problem before, we had hyphens in the path already right?

Reply:
Yeah. I was puzzled. The extra `-exec-2B5CBBC6` broke it when there were no quotes 🤷  Something fishy upstream